luasocket/PSPBUILD

```diff
@@ -5,6 +5,7 @@ pkgdesc="Network support for the Lua language"
 arch=('mips')
 url="https://lunarmodules.github.io/luasocket/"
 license=('MIT')
+groups=('pspdev-default')
 depends=('lua51')
```

Comment:
Does this only work with lua51 or lua in general? If it works with multiple versions, you could depend on lua instead.

Reply:
There's a problem: it will only work with the version of lua it was built with. So, in order to build support multiple versions of lua, it needs to be built multiple times. I am now trying to build a good example that would work with this code.

Reply:
Ah okay, that's fair

Reply:
And yes, `lua51` is not in `pspdev-default` anymore, so this was removed.